src/results_analyze/load_results_package.py

```diff
@@ -371,9 +347,9 @@ def create_package_level_dataframe(self, json_filename=None):
         files_dataframe, metadata_dataframe = self.modify_package_level_dataframe(path_json_dataframe)
 
         # Append metadata level information to a MetaData File
-        self.append_metadata_dataframe(metadata_dataframe)
+        # self.append_metadata_dataframe(metadata_dataframe)
```

Comment:
Not sure if we want to keep these around?

Reply:
No oops, that's a mistake I'll remove that.